images/gitlab/tests/main.tf

```diff
@@ -0,0 +1,93 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digests" {
+  description = "The image digest to run tests over."
+  type = object({
+    kas = string
+  })
+}
+
+// Invoke the image's version command.
+// $IMAGE_NAME is populated with the image name by digest.
+// TODO: Update or remove this test as appropriate.
+data "oci_exec_test" "kas-version" {
+  digest = var.digests.kas
+  script = "docker run --rm $IMAGE_NAME -v"
+}
```

Comment:
I wouldn't bother with this, since the test below is much more comprehensive.

Reply:
Cool, cleaned it up!